### PR TITLE
Add more information to the readme

### PR DESCRIPTION
**Pull request by bedroesb**
- Automation info
- adding mapping rules

### CODE REVIEW

**Review by tabbassidaloii — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,11 +13,29 @@ To align visions and to keep track of content changes in both repositories we cr
...
+
+### Automatic issue creation when new content is added to one of the resources
+
+Due to the changing nature of both resources, we will have to repeat the mapping exercise in the future. To make this job easier, we keep track of changes in both resources using GitHub issues on this repository. Cache files in the `cache` branch keep track of content using the sidebar files of both resources. If a new page are recipe gets added, an issue will be created describing this addition. This GitHub action is run weekly. The `cache` branch should never be deleted!
 
-Changes made to the [`faircookbook_rdmkit_mapping.yml`](https://github.com/elixir-europe/faircookbook-rdmkit/blob/main/faircookbook_rdmkit_mapping.yml) file will be merged in the `main` branch using pull requests and only when at least one member of each editorial team has approved the changes.
 
 ---
```
Comment (tabbassidaloii) [inline README.md:41]: The FAIR cookbook logo cannot be seen on a white background.

Comment (bedroesb) [reply README.md:41]: Could you get me the new logo that is not white? :) I don't find it anywhere, tip for FAIRCookbook: a media kit page. I see many version of the logo going around!

Comment (tabbassidaloii) [reply README.md:41]: @proccaserra could you please point us to the official FAIR cookbook logo?

**Comment by bedroesb** [comment]
@tabbassidaloii I will create an issue about the FAIRCookbook logo where we can keep track of this, and we do not forget!

**Pull request merged**